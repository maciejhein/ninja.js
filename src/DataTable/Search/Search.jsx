import React from 'react'

const Search = ({ onChange }) => {
  return (
    <div className="p-b-1">
      <input
        type="search"
        className="form-control"
        placeholder="SÃ¸g brugere"
        onChange={onChange} />
    </div>
  )
}

export default Search
