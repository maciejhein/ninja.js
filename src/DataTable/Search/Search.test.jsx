import React from 'react';
import { shallow } from 'enzyme';

import Search from './Search';

describe('Search', () => {
  it('should renders correctly', () => {
    const component = shallow(<Search onChange={jest.fn()} />);

    expect(component).toMatchSnapshot();
  });
});