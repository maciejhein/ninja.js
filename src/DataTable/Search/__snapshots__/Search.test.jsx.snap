// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders without crashing 1`] = `
<div
  className="p-b-1"
>
  <input
    className="form-control"
    onChange={[MockFunction]}
    placeholder="Søg brugere"
    type="search"
  />
</div>
`;

exports[`Search should renders correctly 1`] = `
<div
  className="p-b-1"
>
  <input
    className="form-control"
    onChange={[MockFunction]}
    placeholder="Søg brugere"
    type="search"
  />
</div>
`;
